Eastern-Research-Group/csb-rebate-forms-app
# Feature/update 2023 prf school district priority data

## Related Issues:
* CSBAPP-275

## Main Changes:
Update field used in BAP query for 2023 PRF data to get the school district priority data from the 2023 FRF.

## Steps To Test:
1. Ask the BAP to change an existing 2023 FRF submission's status to "Accepted"
2. Create a new 2023 PRF submission.
3. Ensure the "school district priority" field matches data used in the 2023 FRF submission.
